docker/Dockerfile.sgx

```diff
@@ -0,0 +1,44 @@
+From ubuntu:18.04
+
+RUN apt-get update && apt-get install -y \
+     make \
+     libzmq3-dev \
+     libssl1.1 \
+     curl \
+     libprotobuf-dev
+
+ARG SGX_SDK=https://download.01.org/intel-sgx/sgx-linux/2.7.1/distro/ubuntu18.04-server/sgx_linux_x64_sdk_2.7.101.3.bin
```

Comment:
is there a need for SDK? Isn't PSW enough for running?

Reply:
I'll have a test to see

Reply:
yes,  it's not needed. I'll remove it.